File: leetcode Solns/Easy/missing_number.py
# Problem Link:
# https://leetcode.com/problems/missing-number/
#
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        ans=0;
        for i in range(len(nums)):
            ans^=nums[i]
            ans^=i+1
        return ans
        

File: leetcode Solns/Daily Challenges/missingNumber.java
// Problem Link:
// https://leetcode.com/problems/missing-number/
//
class Solution {
    public int missingNumber(int[] nums) {
        int l = nums.length;
        int sum = 0;
        for(int i=0;i<l;i++)
            sum+=nums[i];
        
        return (l*(l+1)/2) - sum;
    }
}